src/main/resources/com/adobe/epubcheck/messages/MessageBundle.properties

```diff
@@ -131,6 +131,8 @@ MED_004=Image file header may be corrupted.
 MED_005=Media Overlay audio reference %1$s to non-standard audio type %2$s found.
 MED_006=Some browsers do not support rendering SVG images which use a filename in the xlink:href property.
 MED_007=Foreign resources can only be referenced from "source" elements with an explicit "type" attribute; found resource "%1$s" of foreign type "%2$s".
+MED_014=A fragment identifier is required.
+MED_015=Empty fragment identifiers are not allowed.
```

Comment:
I'm not sure this warrants two distinct messages, maybe we can combine these into only `MED-014`? ("A (non-empty) fragment identifier is required")
WDYT?